rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -364,8 +364,8 @@ def _parse_raw_user_utterance(self, step: Dict[Text, Any]) -> Optional[UserUtter
         else:
             raw_entities = step.get(KEY_ENTITIES, [])
             entities = self._parse_raw_entities(raw_entities)
+            # set text to `None` for e2e
```

Comment:
then not for e2e, I think the comment is pointless here

Reply:
i thought we added it as an explanation as to why we are setting it to `None` (unlike before)

Reply:
yeah, but this comment doesn't explain anything, just states whatever can be seen in code

Reply:
I'm in two minds here -- if we remove the comment, than we have left why it is `None` unexplained. If I add a full explanation about it being needed during end-to-end training but not prediction, then we confuse the people at this stage because end-ot-end training is non-existent yet

Reply:
what I mean is this exact comment doesn't explain anything more than `text=None` which pretty much covered but the code. Maybe smth like:
```
# set plain_text to None because only intent was provided in the stories
```

Reply:
done!